packages/ssz/test/unit/utils.ts

```diff
@@ -1,5 +1,6 @@
 import {SimpleSSZType,} from "@chainsafe/ssz-type-schema";
-
+// @ts-ignore
+BigInt.prototype.toJSON = function() { return this.toString(); };
```

Comment:
why do we need this here? this doesn't seem right

Reply:
Yes, you need that. JSON doesnt know how to handle bigints.

Reply:
shouldnt we register it somewhere global?

Reply:
where are we converting bigints to json?

Reply:
whenever you are logging some eth2 type?

Reply:
I don't think we actually do this tho.
I'm wary of attaching custom code to globals. if, for some (sad) reason, a library relies on the default behavior, we're effecting that.